Add [all] dependency to clarify napari and Qt binding installation (#56)
napari/napari-plugin-template: README.md

Your plugin template is ready!  Next steps:
1. `cd` into your new directory
    cd napari-growth-cone-finder

>     # you probably want to install your new package into your existing environment
>     # the below command will install the package in editable mode with
>     # napari and Qt bindings
>     pip install -e .[all]

Reviewer: ```suggestion
    # Use the following command to install your package in editable mode, 
    # as well as napari and Qt bindings into your existing environment.
    pip install -e .[all]
```

Reviewer: I would simplify the language here.

Reviewer: If you make this change the same would need to be done in _tasks.py.

Your plugin template is ready!  Next steps:
1. `cd` into your new directory
    cd napari-growth-cone-finder

>     # you probably want to install your new package into your environment

Reviewer: ```suggestion
    # you probably want to install your new package into your existing environment
```
